src/test/testData/StandaloneModeMediumTests/src/quick_fixes/overlapping_quick_fixes.expected.java

```diff
@@ -6,8 +6,8 @@ void m() {
     var file = new File("");
     var array = new int[5];
     if (file != null && (file.isFile() || Arrays.toString(array).equals(""))) {
-      /* ... */
+        /* ... */
+      }
```

Comment:
Do we know why this needs to be changed? Is it a recent change on the analyzer side? If yes, it could be a feedback to make, as the indentation looks off

Reply:
Hi Damien, I created a ticket to investigate this issue

Reply:
Fixed https://github.com/SonarSource/sonarlint-intellij/pull/1043/commits/f5ddcf666f12e4eb46a4957855636b6f0f626997